kairon/shared/chat/processor.py

```
@@ -310,6 +310,43 @@ async def upload_media_to_bsp(bot: str, user: str, channel: str, file_path: str,
+            logger.error(f"Error deleting file to BSP: {str(e)}")
+            raise AppException(f"Failed to delete media: {str(e)}")
+
+    @staticmethod
+    def fetch_media_from_bsp(bot: str, channel: str, media_id: str):
+        from ..channels.whatsapp.bsp.factory import BusinessServiceProviderFactory
+        try:
+            channel_config = ChatDataProcessor.get_channel_config(channel, bot)
+            bsp_type = channel_config.get("config").get("bsp_type", "meta")
+            media_url = BusinessServiceProviderFactory.get_instance(bsp_type).fetch_media_file_url(media_id,
+                                                                                                   channel_config)
+            return media_url
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)